helm-chart/README.md

```diff
@@ -1,7 +1,9 @@
 # Install with Helm
 
 ## Prerequisites
-* Helm: https://helm.sh/docs/intro/install/
+* [Kubernetes Cluster](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/create-cluster-kubeadm/)
```

Comment:
I would link to the more general "Kubernetes setup options" page instead https://kubernetes.io/docs/setup/